### PR TITLE
travis: Build with Ice from downloads.openmicroscopy.org

### PR DESCRIPTION
**Pull request by rleigh-dundee**
The original PR was closed.  This is the travis-specific part of the PR, so it will build with Ice 3.5 from downloads.openmicroscopy.org.

--------

Testing: check travis is green.

### CODE REVIEW

**Comment by joshmoore** [comment]
Since this file "may be used by hand as well" (which at least I do), can we detect `TRAVIS` in `get_ice` and exit if not?

**Comment by rleigh-dundee** [comment]
@joshmoore Should be taken care of now.  Using TRAVIS=true (from https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables), it's now only triggered if this is set.

File: components/tools/OmeroPy/scripts [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 7288f3362beb5a76778e5cd41dc6ef6e0e47e4f7
+Subproject commit 4d96b4f98fcfb88281a9cf0a3c61f9da386d32b8
```
Comment (joshmoore) [inline components/tools/OmeroPy/scripts]: revert?

**Comment by joshmoore** [comment]
> Should be taken care of now. 

ta.

File: components/tools/OmeroPy/scripts (hunk 1 above)
Comment (rleigh-dundee) [inline components/tools/OmeroPy/scripts]: Done.

**Comment by rleigh-dundee** [comment]
@joshmoore Fixed the rebased submodule SNAFU.  It's now green, and good to go I think.

**Comment by joshmoore** [comment]
Thanks, @rleigh-dundee -- a few points:

 * testing `travis-build py-test` locally: :+1:
 * testing `TRAVIS=true py-test` locally: :+1:
 * testing `travis-build` (no arg) found an unrelated bug -- you ok to cherry-pick https://github.com/joshmoore/openmicroscopy/commit/df8d6028e7737111ce09d86a2e2f55fde733a834 ?

**Comment by rleigh-dundee** [comment]
@joshmoore Picked and pushed your fix.

**Comment by joshmoore** [comment]
Merging on green.

**Pull request merged**